data_model/src/trigger.rs

```diff
@@ -317,11 +293,8 @@ pub mod action {
     /// Trait for common methods for all [`Action`]'s
     #[cfg(feature = "transparent_api")]
     pub trait ActionTrait {
```

Comment:
I don't think this trait is needed. But if it is, please move it's definition into `iroha_core`